federated_learning/financial_fraud_detection/README.md

```diff
@@ -48,6 +48,13 @@ In the privacy-enhanced joint machine learning architecture, each financial inst
 
 ## Quick Start
 
+#### Environment requirement
+
+OS version: Ubuntu 20.04
+
+Python version: Python 3.8+.
+
+
```

Comment:
Because we can't ensure the application always works with new versions of Python, can you provide a specific environment? Also, it should include the specs of CPU and system memory. Thank you.

For example,
```suggestion
This application has been tested using Python 3.8 on Ubuntu 20.04 with <CPU Spec> and <XX> GB memory.
```

Reply:
I have made some modifications.